onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -39,6 +39,14 @@
 _MATH_PI = math.pi
 
 
+@torch_op("aten::_local_scalar_dense")
+def aten__local_scalar_dense(self: TTensor) -> TTensor:
```

Comment:
Could you add a docstring with its aten signature from https://aka.ms/native-functions  (looks like `_local_scalar_dense(Tensor self) -> Scalar`. Would be nice to have a reference to its implementation or documentation too.